packages/clerk-js/src/ui/components/SignIn/SignInAccountSwitcher.tsx

```diff
@@ -125,6 +126,4 @@ const SignInAccountSwitcherInternal = () => {
     </Flow.Part>
   );
 };
-export const SignInAccountSwitcher = withRedirectToSignInTask(
```

Comment:
I've removed the `withRedirectToSignInTask` guard from here since the user should be allowed to switch sessions regardless if they have a pending task